src/InvalidArgumentException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Fig\Cache;
```

Comment:
I believe this is unrelated to this PR, but should the vendor and package namespace, for consistency, be

```php
namespace Psr\Cache\Util;
```

?

Consistency always wins, right?

For reference, see [`Psr\Http\Message`](https://github.com/php-fig/http-message/blob/master/src/MessageInterface.php#L3).